CLAUDE.md

```diff
@@ -0,0 +1,59 @@
+# MONDO Ontology Project Guide
+
+## Project Layout
+- Main development file is `src/ontology/mondo-edit.obo`
+- This file is LARGE, never Search or grep this directly EXCEPT using the tools below
+- individual terms checked out in `terms`
+
+## Querying ontology
+
+- For most searches we STRONGLY RECOMMEND using OLS
+    - `runoak -i ols:mondo info "diabetic"`
+- synonyms may be incomplete, if you don't find what you expect, try similar terms
+- To look at a specific term if you know the ID:
+    - `obo-grep.pl -r 'id: MONDO:1234567' srcology/mono-edit.obo`
+- All mentions of an ID
+    - `obo-grep.pl -r 'MONDO:1234567' src/ontology/mondo-edit.obo`
+- DO NOT bother doing your own greps over the file, or looking for other files, unless otherwise asked, you will just waste time.
+- ONLY use the methods above for searching the ontology
+
+## Before making edits
+- Read the request carefully and make a plan, especially if there is nuance
+- If related issues are mentioned read them: `gh issue view GITHUB-ISSUE-NUMBER`
+- Do a background search e.g. using `aurelian websearch "SEARCH TERM"`
+- Retrieve URLs like `aurelian geturl https://en.wikipedia.org/wiki/Apoptosis`
+- Retrieve full text (or abstract if full text not availables): `aurelian fulltext "PMID:19173642"`
+- if a PMID is mentioned in the issue, ALWAYS try and read it
+- ALWAYS check proposed parent terms for consistency
+- For terms that are compositional, check `src/patterns/dosdp-patterns/*yaml`
+
+## Edits
+- Do not edit the edit file directly, it's large
```

Comment:
Do not edit the edit file directly, it's large --> should the second "edit" be mondo-edit.obo?